alf/environments/suite_carla.py

```diff
@@ -385,6 +386,15 @@ def __init__(self,
         else:
             self._bev_sensor = None
 
+        self._with_dynamic_object_sensor = with_dynamic_object_sensor
```

Comment:
Seems that `self._with_dyanmic_object_sensor` is not used.

Reply:
Removed.